src/content/en/tools/chrome-devtools/_book.yaml

```
@@ -0,0 +1,20 @@
+upper_tabs:
+- include: /web/_upper_tabs-Fundamentals.yaml
+- name: Tools
+  heading: Chrome DevTools
+  path: /web/tools/
+  menu:
+  - include: /web/tools/_menu_toc.yaml
+  lower_tabs:
+    other:
+    - name: Guides
+      contents:
+      - include: /web/tools/chrome-devtools/_toc.yaml
+    - name: Blog
+      contents:
+      - include: /web/tools/chrome-devtools/blog/_toc.yaml
+    - name: What's new
+      contents:
+      - include: /web/tools/chrome-devtools/updates/_toc.yaml
```

Same comment as above, you could just link to posts tagged `devtools-whatsnew`

```suggestion
      - title: What's new
        path: /web/updates/tags/devtools-whatsnew
```

---

Same reason with the above. :)